index.html

```diff
@@ -302,15 +302,28 @@ <h3>
       <h2>
         Specifying a powerful feature
       </h2>
-      <p class="advisement">
-        Please register newly specified [=powerful features=] in the
-        [[[powerful-feature-registry]]]. Doing so also provides this Working Group an opportunity
-        to provide feedback and check that integration with this specification is done effectively.
-      </p>
       <p>
-        Each <a>powerful feature</a> has the following permission-related algorithms and types.
-        When the defaults are not suitable for a particular [=powerful feature=], a specification
-        MAY override below algorithms and types below.
+        When a [=specification=] <dfn class="export">specifies a powerful feature</dfn> it:
```

Comment:
Hm, where is `[=specification=]` supposed to be linking to?

Reply:
Going to assume that was an accident and merge - if it was supposed to do something, please send a new PR! 🙈